### DISCUSSION TITLE
Why is my IP banned from Monkeytype?

### DISCUSSION THREAD

**Discussion by Whydidthechickencrosstheroad** [monkeytypegame/monkeytype#5861] [Questions] [answered]
This is the ban I see and I never seen this before. Can you tell me what is this?
![Monkeytype ban](https://github.com/user-attachments/assets/f004ef29-2ca0-41d6-8d4e-7478ba7675a2)

**Comment by Whydidthechickencrosstheroad** [comment]
Note: This is a home IP and I haven't done any violations before

**Comment by Miodec** [comment, marked as answer]
Send an email to support@monkeytype.com

**Reply by Whydidthechickencrosstheroad** [reply]
I did but you haven't answered

**Reply by Miodec** [reply]
Whats your email address?

**Reply by Whydidthechickencrosstheroad** [reply]
Cannot be shared here

**Comment by Whydidthechickencrosstheroad** [comment]
I'm curious how long is the ban, it doesn't say how long by the way. Can someone tell us
